core/src/main/java/com/scalar/db/storage/cosmos/Record.java

```diff
@@ -58,7 +58,7 @@ public String getConcatenatedPartitionKey() {
   }
 
   public Map<String, Object> getValues() {
-    return ImmutableMap.copyOf(values);
+    return Collections.unmodifiableMap(values);
```

Comment:
Just curious. Any reason for this change?

Reply:
Oh sorry, I forgot mentioned it. We need this because the guava `ImmutableMap` doesn't support null values. Putting null values to this `values` map indicates overwriting the existing values with null values.